spec/support/pages/page.rb

```diff
@@ -75,7 +75,6 @@ def view_card(card_name, overlay_class=nil, &block)
     retry_stale_element do
       find('.card-content', text: card_name).click
     end
-    synchronize_content! 'CLOSE'
```

Comment:
Why was this taken out? Was it causing an issue or just not necessary any longer? I think it serves to wait until the card is rendered before allowing the spec to continue.